Automattic/jetpack
# Global Styles: Add Help Link to Plan Limitation Message

Resolves https://github.com/Automattic/wp-calypso/issues/93401

## Proposed changes:

* Adds a new link to the plan limitation notice message on the editor

||Before | After| Clicking the link|
|-------|-------|------|------|
|Site Editor| ![CleanShot 2024-09-05 at 13 35 39@2x](https://github.com/user-attachments/assets/71119645-b84f-4c1d-8fb4-f00e0cca843b) |![CleanShot 2024-09-06 at 11 41 28@2x](https://github.com/user-attachments/assets/1de7caf4-990c-4fd0-9f87-a0e7ed7e4cf8) |      ![CleanShot 2024-09-06 at 11 41 45@2x](https://github.com/user-attachments/assets/51e0913a-98ff-409b-a26e-a5246dc38a2d)       |
|Post Editor| ![CleanShot 2024-09-23 at 18 33 16@2x](https://github.com/user-attachments/assets/33b75ac1-bdf3-49a1-881e-4cde3cae6fdc) |    ![CleanShot 2024-09-23 at 18 31 16@2x](https://github.com/user-attachments/assets/1114bf17-7836-41fa-b8f9-89b88b56a992)       |  ![CleanShot 2024-09-23 at 18 32 46@2x](https://github.com/user-attachments/assets/d65697ab-4f2f-41c9-af21-71f12a0ef585)  |

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply these changes using [the instructions below](https://github.com/Automattic/jetpack/pull/39226#issuecomment-2328576358)
* Navigate to a site that has a lower than Premium plan, e.g. free
* Sandbox your environment. If you are using a Simple Site, sandbox the site
- Navigate to the Styles editor, make sure your site has a theme that allows it, for example Twenty Twenty Four theme. The URL should be `https://<your-site>/wp-admin/site-editor.php?path=%2Fwp_global_styles`
- Change the Style and click on the main editor to open it
- Check the top notice is displayed, and it contains a new link `Learn more`. Click on the link and check the Help Center opens with the URL  https://wordpress.com/support/using-styles/#access-to-styles
- Save your changes
* Navigate to any Post or create a new one
* Check the top notice is displayed, and it contains a new link `Learn more`. Click on the link and check the Help Center opens with the URL  https://wordpress.com/support/using-styles/#access-to-styles
